GOLExtractor/Source/Configuration.cpp

```diff
@@ -0,0 +1,3 @@
+#include "Configuration.h"
+#include "Color.h"
+auto laFabricaDeColor = commonItems::Color::Factory();
```

Comment:
If the only point of Configuration is a place for the definition of this factory, I'd move the definition elsewhere (main.cpp even!) and remove Configuration